src/chart/funnel/FunnelSeries.ts

```diff
@@ -95,6 +97,18 @@ export interface FunnelSeriesOption
     funnelAlign?: HorizontalAlign | VerticalAlign
 
     data?: (OptionDataValueNumeric | OptionDataValueNumeric[] | FunnelDataItemOption)[]
+
+    exitWidth?: string
+
+    showRate?: boolean
+
+    dynamicHeight?: boolean
+
+    /**
+     * @param rateFixed determine how much decimal places rate will keep
+     */
+
+    rateFixed?: number
```

Comment:
It should be  `rateLabel.precision`. And rate should be a value that ranges from 0 to 1.